### ISSUE TITLE
Display column number to make locating compiler errors easier

### ISSUE THREAD

**Issue by sschwarzer** [PMunch/nim-playground-frontend#12]
When I get an error message from the compiler, it contains line and column. However, the column isn't displayed on the page, which would be very helpful to find the location of the error in the editor.

It's not nessary to have a ruler or similar at the top of the editor text area. It would be enough if the column where the cursor is would be displayed above or below the editor text area. While you're at it, you could display line and column together, as found in other editors, e. g. "3, 8" for line 3, column 8, or more explicit as "line 3, column 8".

**Comment by PMunch** [comment]
Good idea, I'll have to see what is possible with the editor we're using though. Will keep it in mind when the site is being redesigned.
